src/apps/front-office/design-system/components/ui/DealsOfTheDay/DealCard.jsx

```diff
@@ -0,0 +1,79 @@
+import { FaShoppingCart } from "react-icons/fa";
+import { FcRating } from "react-icons/fc";
+
+const DealCard = ({
+  imageSrc,
+  title,
+  rating,
+  by,
+  price,
+  oldPrice,
+  timeCards,
+}) => {
+  return (
+    <div className="w-full sm:w-[calc(100%/2-1rem)] lg:w-[calc(100%/3-1rem)] xl:w-[calc(100%/4-1rem)] p-2">
+      <div className="border-0 pb-12 relative bg-white rounded-2xl overflow-hidden transition duration-200 ">
+        <div className="relative z-0">
+          <div className="relative overflow-hidden rounded-3xl">
+            <a className="block" href="#">
```

Comment:
Please, use `link` component and `URLS` object to link the pages instead of writing it statically.